src/main/java/org/terasology/module/inventory/events/RequestInventoryEvent.java

```diff
@@ -3,8 +3,13 @@
 package org.terasology.module.inventory.events;
 
 import org.terasology.engine.entitySystem.event.Event;
+import org.terasology.module.inventory.components.InventoryItemComponent;
+
+import java.util.List;
 
-/**
- */
 public class RequestInventoryEvent implements Event {
+    public List<InventoryItemComponent> items;
+    public RequestInventoryEvent(List<InventoryItemComponent> items) {
```

Comment:
Not sure whether this holds only for network events or in general, but you might need a default constructor without arguments for the automatic deserialization. Might be okay here for now since it is not a network event, though.